fiftyone/operators/utils.py

```
@@ -66,3 +67,40 @@ def is_method_overridden(base_class, sub_class_instance, method_name):
     base_method = getattr(base_class, method_name, None)
     sub_method = getattr(type(sub_class_instance), method_name, None)
     return base_method != sub_method
+
+
+from datetime import datetime, timedelta
```

Ruff

Redefinition of unused `datetime` from line 9

Remove definition: `datetime`

(F811)